samples/type-ahead-search-adaptive-cards/nodejs/Readme.md

```diff
@@ -0,0 +1,70 @@
+---
+page_type: sample
+description: Demonstrating the feature of type ahead search (static and dynamic) control in Adaptive Cards.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+extensions:
+contentType: samples
+createdDate: "24-12-2021 23:30:17"
+---
+
+# Type ahead search control on Adaptive Cards
+
+This sample shows the feature of type ahead search (static and dynamic) control in Adaptive Cards.
+
+`Static search`
+
+![static search card](Images/staticSearchCard.png)
+
+`Dynamic search`
+
+![dynamic search card](Images/dynamicSearchCard.png)
+
+![dynamic search result](Images/dynamicSearchResult.png)
+
+## Prerequisites
+
+- [NodeJS](https://nodejs.org/en/)
+- [ngrok](https://ngrok.com/) or equivalent tunnelling solution
+
+### 1. Setup for Bot
+In Azure portal, create a [Azure Bot resource](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-authentication?view=azure-bot-service-4.0&tabs=csharp%2Caadv2).
+
+- Ensure that you've [enabled the Teams Channel](https://docs.microsoft.com/en-us/azure/bot-service/channel-connect-teams?view=azure-bot-service-4.0)
```

Comment:
Add a screenshot of Teams channel, Ask Prithvi because he has that contents.

Reply:
![image](https://user-images.githubusercontent.com/59052547/147216691-f25329b5-776b-49df-a878-64b12ab6e6c6.png)